src/java.base/share/classes/jdk/internal/foreign/abi/AbstractLinker.java

```diff
@@ -270,15 +278,33 @@ private static FunctionDescriptor stripNames(FunctionDescriptor function) {
                 .orElseGet(() -> FunctionDescriptor.ofVoid(stripNames(function.argumentLayouts())));
     }
 
```

Comment:
Would it make sense to add a "hook" that exotic platforms can use to modify the initialization of the Map?

Reply:
Yes, I meant an internal hook only for "porters".